go-controller/pkg/node/gateway_shared_intf_linux.go

```diff
@@ -98,29 +98,29 @@ func setupLocalNodeAccessBridge(nodeName string, subnets []*net.IPNet) error {
 	return nil
 }
 
-func addSharedGatewayIptRules(service *kapi.Service, nodeIP *net.IPNet) {
-	rules := getGatewayIPTRules(service, "", nodeIP)
+func addSharedGatewayIptRules(service *kapi.Service, nodeIPs []*net.IPNet) {
```

Comment:
sometimes iptables is Ipt and others it is IPT we should pick one and be consistent

Reply:
Good remark, I will go through my patch again and align the naming convention.